main.py

```
@@ -0,0 +1,70 @@
+
+
+## Define a Pydantic model for request/response validation
+# class Item(BaseModel):
+#    name: str
+#    description: Optional[str] = None
+#    price: float
+#    tax: Optional[float] = None
+
+
+class Timeline(BaseModel):
+    data: Dict[str, int] = Field(
```

Ruff (0.8.0)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)